src/features/import/hooks/useColumn.ts

```diff
@@ -56,32 +57,38 @@ export default function useColumn(orgId: number) {
   const nativeFieldsOptions: Option[] = Object.values(NATIVE_PERSON_FIELDS)
     .filter((fieldSlug) => fieldSlug != 'id' && fieldSlug != 'ext_id')
     .map((fieldSlug) => ({
+      disabled: false,
       label: globalMessages.personFields[fieldSlug](),
       value: `field:${fieldSlug}`,
     }));
 
-  const customFieldsOptions: Option[] = customFields
-    .filter(
-      (field) => field.organization.id == orgId || field.org_write == 'suborgs'
-    )
-    .map((field) => {
-      if (field.type === CUSTOM_FIELD_TYPE.DATE) {
-        return {
-          label: field.title,
-          value: `date:${field.slug}`,
-        };
-      } else if (field.type == CUSTOM_FIELD_TYPE.ENUM && field.enum_choices) {
-        return {
-          label: field.title,
-          value: `enum:${field.slug}`,
-        };
-      } else {
-        return {
-          label: field.title,
-          value: `field:${field.slug}`,
-        };
-      }
-    });
+  const customFieldsOptions: Option[] = customFields.map((field) => {
+    if (field.type === CUSTOM_FIELD_TYPE.DATE) {
+      return {
+        disabled: !(
+          field.organization.id == orgId || field.org_write == 'suborgs'
+        ),
+        label: field.title,
+        value: `date:${field.slug}`,
+      };
+    } else if (field.type == CUSTOM_FIELD_TYPE.ENUM && field.enum_choices) {
+      return {
+        disabled: !(
+          field.organization.id == orgId || field.org_write == 'suborgs'
+        ),
+        label: field.title,
```

Comment:
The open question in the issue asks whether it may be valuable to indicate _why_ an option is disabled. I'm thinking that maybe this label could be a good opportunity for that. What do you think about appending `' (read-only)'` or something like that (internationalized of course) for the read-only options?

Reply:
Updated the image in the PR with this, helps making it clearer why they are disabled I think.